### DISCUSSION TITLE
How to Reset or Manage Realtime Connections Limit on Free Plan?

### DISCUSSION THREAD

**Discussion by PinsaraPerera** [supabase/supabase#31123] [Questions] [answered]
I'm currently using the free plan, and I understand there's a limit of 200 peak realtime connections. While testing the product I'm working on, I somehow hit the connection cap quite quickly. Now, realtime database listening has stopped working altogether.

I have a few questions:

01) Is there a way to manually reset the realtime connections without upgrading to a higher plan?
02) How often does the cap reset automatically?

Additionally, I’ve checked my implementation and ensured that I’m unsubscribing from channels when they’re no longer needed, but I’m still hitting the cap during testing.

**Comment by GaryAustin1** [comment, marked as answer]
The 200 limit is a point in time limit.  When you are below 200 it should work again.

**Reply by PinsaraPerera** [reply]
Thank you very much. yes its now work again. Still showing the warning "Exceeding usage limits" but it works now.

**Reply by GaryAustin1** [reply]
Yeah, I don't know how the track the exceeding for the month, it may stay awhile so you know it exceeded as otherwise all you get are errors to the clients when it is above 200.
